library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -153,18 +160,38 @@ class Client() {
         return Client(address = address, privateKeyBundleV1 = bundle, apiClient = apiClient)
     }
 
-    fun create(account: SigningKey, options: ClientOptions? = null): Client {
+    fun create(
+        account: SigningKey,
+        options: ClientOptions? = null,
+    ): Client {
         val clientOptions = options ?: ClientOptions()
         val apiClient =
             GRPCApiClient(environment = clientOptions.api.env, secure = clientOptions.api.isSecure)
-        return create(account = account, apiClient = apiClient, options = options)
+        return create(
+            account = account,
+            apiClient = apiClient,
+            options = options,
+        )
     }
 
-    fun create(account: SigningKey, apiClient: ApiClient, options: ClientOptions? = null): Client {
+    fun create(
+        account: SigningKey,
+        apiClient: ApiClient,
+        options: ClientOptions? = null,
+    ): Client {
         return runBlocking {
             try {
                 val privateKeyBundleV1 = loadOrCreateKeys(account, apiClient, options)
-                val client = Client(account.address, privateKeyBundleV1, apiClient)
+                val libXMTPClient: FfiXmtpClient =
+                    createClient(
+                        logger = logger,
+                        ffiInboxOwner = account,
+                        host = "https://dev.xmtp.network:5556",
+                        isSecure = true,
+                        db = null,
+                        encryptionKey = null
+                    )
```

Comment:
Should I be feature flagging this creation? Doesn't seem harmful to do this behind the scenes for all existing clients or is it?

Reply:
The only harm is that it will prompt for a signature in some cases.

Reply:
Also since production won't be available until beta we could wind up with mismatches between dev and prod (prod identity used in dev)